ui/src/app/settings/views/Configuration/DeployFormFields/DeployFormFields.test.tsx

```diff
@@ -61,15 +61,54 @@ describe("DeployFormFields", () => {
     });
   });
 
-  it("can render", () => {
+  it("displays the deploy configuration form with correct fields", () => {
     const store = mockStore(state);
-    const wrapper = mount(
+    render(
       <Provider store={store}>
         <MemoryRouter initialEntries={[{ pathname: "/", key: "testKey" }]}>
           <DeployForm />
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find("DeployFormFields").exists()).toBe(true);
+
+    const form = screen.getByRole("form", { name: "deploy configuration" });
+    expect(form).toBeInTheDocument();
+
+    expect(
+      within(form).getByRole("combobox", {
+        name: /Default operating system used for deployment/,
```

Comment:
I think it might be better to follow the `Label` enum pattern we've been using so that these tests don't fail on content changes. e.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagFormChanges/TagFormChanges.tsx#L29

Reply:
Thanks! I'll look into all these label-related comments separately, as part of a single task: https://github.com/canonical-web-and-design/app-tribe/issues/797